lib/logstash/filters/uuid.rb

```diff
@@ -45,13 +45,12 @@ def filter(event)
     return unless filter?(event)
 
     if overwrite
-      event[target] = SecureRandom.uuid
+      event[target] = SecureRandom.uuid.force_encoding("UTF-8")
```

Comment:
Hi, is this force_encoding really necessary? forcing encoding should be always the last way to archive this.